install_config/topics/ocp_osp_admin_tasks.adoc

```diff
@@ -1,7 +1,7 @@
 [[osp_accounts]]
 ==== Creating OpenStack User Accounts, Projects and Roles
 
-Before installing {product-title}, the Red Hat OpenStack Platform (RHOSP) 
+Before installing {product-title}, the Red Hat OpenStack Platform (RHOSP)
 environment requires a project, often referred to as a _tenant_,
 that stores the OpenStack instances that are to install the {product-title}. This project
```

Comment:
This project
requires ownership by a user, and the role of that user to be set to `\_member_`.